lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1163,6 +1176,8 @@ def deploy_vm(self):
 
         # find matching folders
         if self.params['folder'].startswith('/'):
+            if not self.params['folder'].startswith('/vm'):
```

Comment:
okay, i'm just waiting for answer on other points and the uppercase issue and then i will merge it